webpack/scenes/RedHatRepositories/components/Search.js

```diff
@@ -49,11 +49,18 @@ class RepositorySearch extends Component {
 
     if (this.state.searchList.key === 'enabled') {
       params.enabled = true;
+      return {
```

Comment:
Is this an equivalent?

```
const keyEndpointMap = {
 'enabled': /repositories/auto_complete_search',
 'available': '/repository_sets/auto_complete_search',
};

return {
    endpoint: keyEndpointMap[this.state.searchList.key],
    params,
};
```

Reply:
Oh, I didn't see the `params.enabled = true` part so that'd still need to be checked

Reply:
Basically, I'd just like to see refactored to 1 return statement if possible, so maybe this is another option:

```
if (available)
 endpoint = 'something';
else if (enabled)
  params.enabled = true;
  endpoint = 'something else';
}

return { endpoint: endpoint, params };
```
```